### PR TITLE
Add version number to extension's menu

### PR DESCRIPTION
**Pull request by UnderscoreX5**
Dinamically takes the manifest's version number and displays it under the credits text. **Full compatibility with Firefox and Safari is the current priority.** Placement of the number must be changed to be in the bottom right corner. Currently broken i guess.

### CODE REVIEW

File: Extensions/combined/popup.html [hunk 1]
```diff
@@ -13,6 +13,7 @@
       <svg width="48" viewBox="0 0 24 24"><path d="M14.9 3H6c-.9 0-1.6.5-1.9 1.2l-3 7c-.1.3-.1.5-.1.7v2c0 1.1.9 2 2 2h6.3l-.9 4.5c-.1.5 0 1 .4 1.4l1.1 1.1 6.5-6.6c.4-.4.6-.9.6-1.4V5c-.1-1.1-1-2-2.1-2zm7.4 12.8h-2.9c-.4 0-.7-.3-.7-.7V3.9c0-.4.3-.7.7-.7h2.9c.4 0 .7.3.7.7V15c0 .4-.3.8-.7.8z" fill="red"/><path d="m8 12.5 5.1-2.9L8 6.7v5.8z" fill="#fff"/></svg>
       <h1>Return YouTube Dislike</h1>
       <p>by Dmitrii Selivanov & Community</p>
+      <p>v2.0.0.3</p>
```
Comment (himanshudabas) [inline Extensions/combined/popup.html:16]: instead of hardcoding, you might wanna use `chrome.runtime.getManifest().version;`

**Review by himanshudabas — CHANGES_REQUESTED** [review]

**Comment by himanshudabas** [comment]
cc https://github.com/Anarios/return-youtube-dislike/issues/350

File: Extensions/combined/popup.html (hunk 1 above)
Comment (Anarios) [reply Extensions/combined/popup.html:16]: and not chrome, but cross-browser version

Comment (UnderscoreX5) [reply Extensions/combined/popup.html:16]: Yeah that's better. Will change it as soon as I can.

Comment (UnderscoreX5) [reply Extensions/combined/popup.html:16]: I know that the namespace for firefox and safari is 'browser' but is there a cross-browser one fir both chromium and firefox/safari?

Comment (himanshudabas) [reply Extensions/combined/popup.html:16]: this is how we do this in bg-script:

https://github.com/Anarios/return-youtube-dislike/blob/main/Extensions/combined/ryd.background.js#L11

try to test it in both browsers.

Comment (UnderscoreX5) [reply Extensions/combined/popup.html:16]: It shouldn't work or at least not as efficiently, perhaps implementing it this way is better, coming up in seconds with a possible fix.

Comment (UnderscoreX5) [reply Extensions/combined/popup.html:16]: @himanshudabas can you propose the 'chrome.runtime.getManifest().version;' change you did so I can double check on my firefox as well?

Comment (himanshudabas) [reply Extensions/combined/popup.html:16]: popup.html

```html
<p id="ext-version"></p>
```

popup.js

```javascript
function initConfig() {
  initializeVersionNumber();
  ...
}

...

function initializeVersionNumber() {
  const version = chrome.runtime.getManifest().version;
  document.getElementById('ext-version').innerHTML = 'v' + version;
}
```

**Comment by Anarios** [comment]
> @Anarios I just tested chrome.runtime.getManifest().version; in firefox. I don't know why but it works (:

I think it works in later FF versions, but might break in older ones, which would otherwise work. I'm not sure though.

**Comment by UnderscoreX5** [comment]
I suppose that the best way is then to use both browser and chrome namespaces just to be sure it works on all browsers (it works on firefox but what about safari?). Will try something tomorrow after work.

**Comment by Anarios** [comment]
I think what we do in background.js can work here as well

**Comment by UnderscoreX5** [comment]
Yeah it probably works just gotta translate it to html tho

File: Extensions/combined/popup.html (hunk 1 above)
Comment (UnderscoreX5) [reply Extensions/combined/popup.html]: I implemented this for now.

**Comment by UnderscoreX5** [comment]
The current version doesn't work on my firefox but I think I've misplaced the function.

**Review by Anarios — APPROVED** [review]

**Pull request merged**